scripts/prohibit-properties-unused-keys

```diff
@@ -8,14 +8,21 @@ source "$self_dir"/utils.sh
 bundle=frontend/base/src/main/resources/GitMacheteBundle.properties
 [[ -f $bundle ]] || die "Bundle file ($bundle) has been (re)moved"
 
-exit_code=0
-for key in $(
-  grep -Po '.*(?==)' -- $bundle \
-  | grep -Evx 'action\.GitMachete\.[A-Za-z]+\.(description|GitMachete(ContextMenu|Toolbar).text|text)'
-); do
-  if ! git grep -qF "$key" -- '*.java' '*.kt' ":!$bundle"; then
```

Comment:
`git grep` executed for each property key separately... Not Good ™️

Reply:
https://virtuslab.github.io/tips/#regex/lookbehind-lookahead, IMO the only advanced regex feature (assuming that capturing groups aren't considered advanced) you'll ever need